website/src/controllers/user.js

```diff
@@ -606,6 +600,7 @@ api.batchUpdate = function(req, res, next) {
           return cb(code+": "+ (data.message ? data.message : data.err ? data.err : JSON.stringify(data)));
         return cb();
       };
+      if(!api[_req.op]) { return cb(_req.op + ' operation not found'); }
```

Comment:
@crookedneighbor it's fine but due to this line https://github.com/crookedneighbor/habitrpg/blob/batch_update_adjustment/website/src/controllers/user.js#L617 any error is treated as an internal server error. But I think that passing a different error code to the response woul break the site so nevermind, we'll fix it later